lib/posix/posix_other.nim

```diff
@@ -140,12 +140,15 @@ type
     ## used for block sizes
   Clock* {.importc: "clock_t", header: "<sys/types.h>".} = int
   ClockId* {.importc: "clockid_t", header: "<sys/types.h>".} = int
-  Dev* {.importc: "dev_t", header: "<sys/types.h>".} = int
+  Dev* {.importc: "dev_t", header: "<sys/types.h>".} = uint64
+    # WAS: int. Probably not correct but more correct than previous definition
+    # https://lists.debian.org/debian-mips/2011/11/msg00006.html
```

Comment:
I do not appreciate these kinds of comments. In my opinion they are just visual noise with distracting but non-informative. They should be part of the PR description. But they should not be part of the source code.

Reply:
I'd deprecate `posix_other` if this PR got in to be fair. These types are opaque as far as POSIX is concerned, and you can't just copy Linux's definition and call it a day. Haiku for example uses an int32 here.

Reply:
maybe importc types need to be revisited, just look at my discussion with @Araq here https://github.com/nim-lang/zip/pull/42#pullrequestreview-389772201

Reply:
fixed by deferring stdlib changes to https://github.com/nim-lang/Nim/pull/13941